### PR TITLE
feat(admin): add approve and notice init

### PR DESCRIPTION
**Pull request by Muzry**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:

- feat(admin): add approve and notice init

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: apistructs/approve.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ type ApproveDeleteResponse struct {
 // ApproveDetailResponse GET /api/approves/{approveId} 审批详情响应结构
 type ApproveDetailResponse struct {
 	Header
-	ApproveDTO `json:"data"`
+	Data ApproveDTO `json:"data"`
```
Comment (Effet) [inline apistructs/approve.go:82]: i dont know what purpose to changes this and will it break api compatibility?

**Review by Effet — CHANGES_REQUESTED** [review]

File: apistructs/approve.go (hunk 1 above)
Comment (Muzry) [reply apistructs/approve.go:82]: if I keep the struct, the API with a `bundle` that already return the same struct to the upper layer will respond like this:
```json
{
    "success": true,
    "data": {
        "success": true,
        "err": {
            "code": "",
            "msg": "",
            "ctx": null
        },
        "data": {
            "id": 1,
            "orgId": 2,
            "entityId": 0,
            "targetId": 2,
            "targetName": "erda",
            "type": "unblock-application",
            "extra": {
                "appIDs": "29",
                "end": "2021-05-12T17:50:35+08:00",
                "start": "2021-05-12T14:50:35+08:00"
            },
            "title": "erda项目，申请在时间段(2021-05-12 14:50:35~2021-05-12 17:50:35)内的部署权限",
            "priority": "",
            "desc": "aa",
            "status": "pending",
            "submitter": "2",
            "approver": "2",
            "approvalTime": "2021-05-12T17:50:56+08:00",
            "createdAt": "2021-05-12T17:50:56+08:00",
            "updatedAt": "2021-06-30T16:45:13+08:00"
        }
    },
    "err": {
        "code": "",
        "msg": "",
        "ctx": null
    }
}
```
I don't think it meets our expectations

Comment (Effet) [reply apistructs/approve.go:82]: okay

**Review by Effet — APPROVED** [review]

**Pull request merged**